### PR TITLE
Adding extern crate oasis_test to erc20

### PR DESCRIPTION
**Pull request by vishwa-raman**
Added `extern crate oasis-test` to the `test` mod.

### CODE REVIEW

File: examples/erc20/src/main.rs [hunk 1]
```diff
@@ -224,6 +224,10 @@ fn main() {
 
 #[cfg(test)]
 mod tests {
+    // This is required even in Rust 2018. If omitted, rustc will not link in the testing
+    // library and will produce a giant error message.
```
Comment (nhynes) [inline examples/erc20/src/main.rs]:
```suggestion
```

File: examples/erc20/src/main.rs [hunk 2]
```diff
@@ -224,6 +224,10 @@ fn main() {
 
 #[cfg(test)]
 mod tests {
+    // This is required even in Rust 2018. If omitted, rustc will not link in the testing
```
Comment (nhynes) [inline examples/erc20/src/main.rs]:
```suggestion
```

Comment (vishwa-raman) [reply examples/erc20/src/main.rs]: Done.

File: examples/erc20/src/main.rs (hunk 1 above)
Comment (vishwa-raman) [reply examples/erc20/src/main.rs]: Done.

**Pull request merged**